include/blast/system/Restrict.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
```

Comment:
Does `__restrict` really make a difference? Or do you do it because Blaze had it as well?

Reply:
I did not check, but it might. I think Blaze uses it for a reason.